docs/guides/global-lustre/readme.md

```diff
@@ -0,0 +1,79 @@
+---
+authors: Blake Devcich <blake.devcich@hpe.com>
+categories: provisioning
+---
+
+# Global Lustre
+
+## Background
+
+Adding global lustre to rabbit systems allows access to external file systems. This is primarily
+used for Data Movement, where a user can perform `copy_in` and `copy_out` directives with global
+lustre being the source and destination, respectively.
+
+Global lustre fileystems are represented by the `lustrefilesystems` resource in Kubernetes:
+
+```shell
+$ kubectl get lustrefilesystems -A
+NAMESPACE   NAME       FSNAME   MGSNIDS          AGE
+default     mylustre   mylustre 10.1.1.113@tcp   20d
+```
+
+An example resource is as follows:
+
+```yaml
+apiVersion: lus.cray.hpe.com/v1alpha1
+kind: LustreFileSystem
+metadata:
+  name: mylustre
+  namespace: default
+spec:
+  mgsNids: 10.1.1.100@tcp
+  mountRoot: /p/mylustre
+  name: mylustre
+  namespaces:
+    default:
+      modes:
+        - ReadWriteMany
+```
+
+## Namespaces
+
+Note the `spec.namespaces` field. For each namespace listed, the `lustre-fs-operator` creates a
+PV/PVC pair in that namespace. This allows pods in that namespace to access global lustre.
+
+It is recommended to create `lustrefilesystem` resources in the `default` namespace (i.e.
+`metadata.namespace`). Adding `lustrefilesystems` to the `nnf-dm-system` namespace can cause issues
```

Comment:
I think you can drop the "can cause issues" sentence. Just tell the reader how we want them to do it, not how we don't want them to do it.

Reply:
I reworded this.